pkg/gds/emails/client.go

```diff
@@ -116,9 +116,10 @@ func (m *EmailManager) SendVerifyContacts(vasp *pb.VASP) (sent int, err error) {
 // SendVerifyContact sends a verification email to a contact.
 func (m *EmailManager) SendVerifyContact(vasp *pb.VASP, contact *pb.Contact) (err error) {
 	ctx := VerifyContactData{
-		Name:    contact.Name,
-		VID:     vasp.Id,
-		BaseURL: m.conf.VerifyContactBaseURL,
+		Name:        contact.Name,
+		VID:         vasp.Id,
+		BaseURL:     m.conf.VerifyContactBaseURL,
+		DirectoryID: m.conf.DirectoryID,
```

Comment:
Adding this directory ID from the configuration should be enough information for the BFF to route the request to the correct GDS.

Reply:
Got it - and `ID` here means the string `testnet` or `mainnet`? Or the urls `trisatest.net` & `vaspdirectory.net`?

Reply:
In this case it will be `trisatest.net` or `vaspdirectory.net`.